bundle-workflow/src/test_workflow/integ_test/integ_test_suite.py

```diff
@@ -85,7 +86,7 @@ def _execute_integtest_sh(self, endpoint, port, security):
             self.component.name, self.repo.dir
         )
         if os.path.exists(script):
-            cmd = f"sh {script} -b {endpoint} -p {port} -s {str(security).lower()}"
+            cmd = f"bash {script} -b {endpoint} -p {port} -s {str(security).lower()} -v {self.bundle_manifest.build.version}"
```

Comment:
You should be executing `script` with the correct shebang, that needs a `chmod 755`.

Reply:
Sure, the scripts in plugin repo are already w/ 755 permissions. I have updated the default integtest.sh w/ 755 as well.

Reply:
This line shouldn't start with bash. The shebang inside the .sh file defines what shell to use.